### DISCUSSION TITLE
SteamOS 3.6.19 stable - Sunshine no longer allows connections via  Moonlight

### DISCUSSION THREAD

**Discussion by s0th1s** [LizardByte/community#545] [Sunshine]
Anyone else having issues after the latest Stable SteamOS update getting Sunshine to work with Moonlight.  I have noted that Sunshine seems to be running and I can log into the webgui, but cannot connect a client to to Sunshine.

Anyone have any ideas? Is this a bug or something??

**Comment by ReenigneArcher** [comment]
Maybe the firewall was reset with the latest update?

[a comment by an automated account is not shown]
